### PR TITLE
nonSCL route53 0.3.2

### PR DESCRIPTION
**Pull request by dLobatog**
As a dependency of smart_proxy_dns_route53

### CODE REVIEW

File: rubygem-route53/rubygem-route53.spec [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        %{buildroot}%{gem_dir}/
+
+mkdir -p %{buildroot}%{_bindir}
+cp -pa .%{_bindir}/* \
+        %{buildroot}%{_bindir}/
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
+
+%files
+%dir %{gem_instdir}
+%{_bindir}/route53
```
Comment (dLobatog) [inline rubygem-route53/rubygem-route53.spec]: This is supposedly wrong by https://fedoraproject.org/wiki/Packaging:Guidelines#No_inclusion_of_pre-built_binaries_or_libraries - not sure if the proxy uses the binary directly, so maybe it's best to exclude it?

File: rubygem-route53/rubygem-route53.spec [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+%global gem_name route53
+
+Name: rubygem-%{gem_name}
+Version: 0.3.2
+Release: 1%{?dist}
+Summary: Library for Amazon's Route 53 service
+Group: Development/Languages
+License: GPLv3
+URL: http://github.com/pcorliss/ruby_route_53
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
```
Comment (domcleal) [inline rubygem-route53/rubygem-route53.spec]: Needs to be http for our tests (looks like both non-SCL templates need fixing).

File: comps/comps-foreman-plugins-fedora24.xml [hunk 3]
```diff
@@ -81,6 +81,7 @@
       <packagereq type="default">rubygem-logify</packagereq>
       <packagereq type="default">rubygem-newt</packagereq>
       <packagereq type="default">rubygem-satyr</packagereq>
+      <packagereq type="default">rubygem-route53</packagereq>
```
Comment (domcleal) [inline comps/comps-foreman-plugins-fedora24.xml]: Not alphabetical, ditto on the others.

File: rubygem-route53/rubygem-route53.spec (hunk 2 above)
Comment (dLobatog) [inline rubygem-route53/rubygem-route53.spec]: I'll do that here too if you dont mind

**Comment by dLobatog** [comment]
Updated, including the gem2rpm templates

File: rubygem-route53/rubygem-route53.spec (hunk 1 above)
Comment (domcleal) [inline rubygem-route53/rubygem-route53.spec]: It isn't a pre-built binary, it's a bin stub script that's generated by `gem install`.  The guideline is forbidding binaries/libraries that are already built in the source tarball/gem.  It's fine as-is.

File: rubygem-route53/rubygem-route53.spec [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        %{buildroot}%{_bindir}/
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
+
+%files
+%dir %{gem_instdir}
+%{_bindir}/route53
+%exclude %{gem_instdir}/.gitignore
+%{gem_instdir}/.rspec
+%{gem_instdir}/.ruby-gemset
+%{gem_instdir}/.ruby-version
```
Comment (domcleal) [inline rubygem-route53/rubygem-route53.spec]: All of these dot-files can be excluded, either .* or individually

**Comment by dLobatog** [comment]
Updated to exclude dotfiles from the package

**Comment by domcleal** [comment]
Could you rebase this PR please?

**Comment by domcleal** [comment]
Merged and building, thanks @dLobatog.

**Pull request merged**